administrator/language/en-GB/en-GB.com_csp.ini

```
@@ -4,6 +4,7 @@
 ; Note : All ini files need to be saved as UTF-8
 
 COM_CSP="Content Security Policy"
+COM_CSP_COLLECTING_TRASH_WARNING="You are running your rules in the collect mode. Please note that there are trashed items that don't get colleted here again untill fully deleted."
```

its "detect" not "collect" mode ;)

```suggestion
COM_CSP_COLLECTING_TRASH_WARNING="The Content Security Policy is in detect mode. Items that have been trashed will not be detected again until they are removed from the trash."
```